src/main/java/data/plans/PlanList.java

```diff
@@ -480,4 +488,47 @@ public void deletePlanContainsDeletedWorkout(String workoutToCheck) throws Array
             deletePlan(Integer.toString(planWithDeletedWorkout.get(i) - i));
         }
     }
+
+    public void updatePlanContainsUpdatedWorkout(String workoutToCheck, Workout updatedWorkout) throws
+            InvalidPlanException {
+        ArrayList<Integer> planWithUpdatedWorkout = findPlanContainsTargetWorkout(workoutToCheck);
+        if (planWithUpdatedWorkout.size() <= 0) {
+            return;
+        }
+
+        for (int planNumber : planWithUpdatedWorkout) {
+            System.out.println("\t" + getPlansDisplayList().get(planNumber - 1));
+            assert (checkPlanIndexIsWithinRange(planNumber)) : "Plan number is out of range.";
+        }
```

Comment:
Am I right to assume that this (enhanced) `for` loop is meant to print the plans with the outdated workout?

If so, perhaps consider printing an additional statement prior to this `for` loop to indicate what these plans are for.

For example: "The following plans contains the workout that was just changed and will be updated:"